meta-buildpacks/nodejs/buildpack.toml

```diff
@@ -54,6 +54,11 @@ optional = true
 id = "heroku/nodejs-engine"
 version = "1.1.7"
 
+[[order.group]]
+id = "heroku/nodejs-corepack"
```

Comment:
@colincasey This addition to the buildpack order is missing from `meta-buildpacks/nodejs/CHANGELOG.md`